### PR TITLE
feat: add example launch

### PR DESCRIPTION
**Pull request by wep21**
I added minimal_pub_sub.launch.xml into example package.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,13 @@ ros2 run examples_rclrs_minimal_pub_sub minimal_publisher
 . ./install/setup.sh
 ros2 run examples_rclrs_minimal_pub_sub minimal_subscriber
 ```
+or
+
+```
```
Comment (nnmm) [inline README.md:61]:
```suggestion
```shell
```

**Comment by nnmm** [comment]
Nice! Just one nitpick :)

**Comment by nnmm** [comment]
Although – don't most people use Python launch files nowadays? Do you know if they are always preferred @esteve @tylerjw @jhdcs?

**Comment by tylerjw** [comment]
We use only python launch files but it isn't because we think we need more imperative launch files. We all much prefer the xml launch system from ros1, I just don't think we've learned to write ros2 xml files.

**Review by nnmm — APPROVED** [review]
Yep, I agree, the XML version is nice and short. We could always additionally provide Python or YAML launch files if we want to.

I'll wait a couple of hours until merging though, to give others the chance to leave a comment if they want to.

**Comment by esteve** [comment]
In general I prefer the Python, but for a simple case like this, XML is much cleaner. Thanks @wep21 !

**Review by esteve — APPROVED** [review]

**Pull request merged**